src/content/docs/es/guides/prefetch.mdx

```diff
@@ -32,11 +32,12 @@ La configuración `prefetch` también acepta un objeto de opciones para personal
 
 ### Estrategias de precarga
 
-Astro admite 3 estrategias de precarga para varios casos de uso:
+Astro admite 4 estrategias de precarga para varios casos de uso:
 
 - `hover` (por defecto): Precarga cuando pasas el cursor o te enfocas en el enlace.
 - `tap`: Precarga justo antes de hacer clic en el enlace.
 - `viewport`: Precarga cuando los enlaces entran en el viewport.
+- `load`: Precargar todos los enlaces de la página después de que la página se cargue.
```

Comment:
Not native Spanish speaker, but pretty sure the verb tense here should be this one instead.
```suggestion
- `load`: Precarga todos los enlaces de la página después de que la página se cargue.
```